scrapy/templates/project/module/middlewares.py.tmpl

```diff
@@ -39,7 +39,7 @@ class ${ProjectName}SpiderMiddleware(object):
         # Called when a spider or process_spider_input() method
         # (from other spider middleware) raises an exception.
 
-        # Should return either None or an iterable of Response, dict
+        # Should return either None or an iterable of Request, dict
```

Comment:
Good catch. Since you're at it, could you maybe add `Item` as well? And perhaps mention that any subclass would work too.

Reply:
Item is on the next line (that got me too). And I think the subclass part is implicit. Otherwise, there are lots of places in code and documentation that need updating :)

Reply:
Ouch, the next line! 🤦‍♂️ 
Nevermind then, sorry for the noise.